### PR TITLE
Add grafeas-client library

### PR DESCRIPTION
**Pull request by dazuma**
Create `grafeas-client` library.

Initial synth of `grafeas-client`. The library name is deliberately chosen because the desired namespace is `Grafeas` rather than `Google::Cloud::Grafeas`, but the gem name `grafeas` is currently occupied. This also includes a fair amount of synth hackery to get around a few assumptions being made in artman that generated libraries are in the `google/cloud` namespace.

Also includes a handwritten acceptance test for `grafeas-client`, and Kokoro configs for the library.

### CODE REVIEW

**Comment by blowmage** [comment]
Failed linux build is security_center and unrelated.

File: grafeas-client/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+3. [Enable the Container Analysis API.](https://console.cloud.google.com/apis/library/grafeas.googleapis.com)
+4. [Setup Authentication.](https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+
+### Installation
+```
+$ gem install grafeas
```
Comment (quartzmo) [inline grafeas-client/README.md]: Shouldn't this be `gem install grafeas-client`?

File: grafeas-client/lib/grafeas.rb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+# steps:
+#
+# 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+# 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+# 3. [Enable the Container Analysis API.](https://console.cloud.google.com/apis/library/grafeas.googleapis.com)
+# 4. [Setup Authentication.](https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+#
+# ### Installation
+# ```
+# $ gem install grafeas
```
Comment (quartzmo) [inline grafeas-client/lib/grafeas.rb]: Shouldn't this be `gem install grafeas-client`?

File: grafeas-client/lib/grafeas/v1.rb [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  # steps:
+  #
+  # 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+  # 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+  # 3. [Enable the Container Analysis API.](https://console.cloud.google.com/apis/library/grafeas.googleapis.com)
+  # 4. [Setup Authentication.](https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+  #
+  # ### Installation
+  # ```
+  # $ gem install grafeas
```
Comment (quartzmo) [inline grafeas-client/lib/grafeas/v1.rb]: Shouldn't this be `gem install grafeas-client`?

File: grafeas-client/.rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - "grafeas-client.gemspec"
+    - "lib/**/*"
+    - "Rakefile"
+    - "test/**/*"
+    - "acceptance/**/*"
```
Comment (quartzmo) [inline grafeas-client/.rubocop.yml]: Please add: `TargetRubyVersion: 2.2`

File: grafeas-client/README.md (hunk 1 above)
Comment (dazuma) [reply grafeas-client/README.md]: Yes, good catch!

**Pull request merged**